### DISCUSSION TITLE
Outline on SQLite

### DISCUSSION THREAD

**Discussion by losfair** [outline/outline#4292] [Show and tell]
Hi community 👋

I got Outline working on SQLite and have been running it for a while. Not sure whether the maintainers are willing to accept a SQLite support patch, so sharing my fork here: https://github.com/mvsqlite-ports/outline

So far it's working pretty well for me - the only thing that doesn't work is search. That requires porting search to use SQLite FTS, which isn't done yet.

The reason for using SQLite is that I want to reuse my existing data infrastructure (FoundationDB + mvSQLite) instead of operating another Postgres database.

**Comment by zicklag** [comment]
Thanks for sharing @losfair. I just want to mention that I would use outline on sqlite if it were officially supported. ❤️

**Reply by thedarkside** [reply]
Me as well. Nowadays with all those abstraction layers ready and opensource it shouldn't be that difficult. Sqlite is also pretty stable for small to medium sized deployments and performant on SSD/nvme. Ruby on Rails propagates this for production as well. 

Why isn't it supported?
